app/controllers/api/abstract_controller.rb

```diff
@@ -11,6 +11,12 @@ class AbstractController < ApplicationController
       sorry "Can't find #{exc.klass}(s) with ID(s): #{exc.params}", 404
     end
 
+    rescue_from Mongoid::Errors::InvalidFind do |exc|
```

Comment:
Unused block argument - `exc`. You can omit the argument if you don't care about it.